content/_code-samples/freeze/py/enable_no_freeze.py

```diff
@@ -0,0 +1,29 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountSet, AccountSetFlag
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.wallet import Wallet, generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # connect to testnet
+
+
+# generate wallet
+sender_wallet = generate_faucet_wallet(client=client)
+
+# build accountset transaction to disable freezing
+accountset = AccountSet(account=sender_wallet.classic_address, set_flag=AccountSetFlag.ASF_NO_FREEZE)
+
+# sign transaction
```

Comment:
Before we send here, we should print out to let the user know the code is working (but will take a couple seconds)
E.g.
```
print("Generated wallet, now sending an AccountSet transaction to set the ASF_NO_FREEZE flag...")
```

Reply:
this is not resolved either

Reply:
oh, ok. I didn't do as @JST5000 suggested.  I split the message to make it easier to read
![image](https://user-images.githubusercontent.com/47371105/202363060-c4b56704-db00-4bd6-a471-bf89bdf01b09.png)

Reply:
oh, ok. I didn't do as @JST5000 suggested. I split the message to make it understandable 

<img width="701" alt="Screenshot 2022-11-17 062006" src="https://user-images.githubusercontent.com/47371105/202363207-62f49a68-d78d-45b6-99a0-ae7d9a81ae58.png">